src/main/java/org/terasology/potions/PotionStatusUISystem.java

```diff
@@ -13,115 +13,117 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
 package org.terasology.potions;
 
-import org.terasology.entitySystem.entity.EntityManager;
+import org.terasology.engine.Time;
 import org.terasology.entitySystem.entity.EntityRef;
 import org.terasology.entitySystem.event.EventPriority;
 import org.terasology.entitySystem.event.ReceiveEvent;
 import org.terasology.entitySystem.systems.BaseComponentSystem;
 import org.terasology.entitySystem.systems.RegisterMode;
 import org.terasology.entitySystem.systems.RegisterSystem;
-import org.terasology.input.ButtonState;
-import org.terasology.input.binds.general.PauseButton;
 import org.terasology.input.binds.inventory.InventoryButton;
 import org.terasology.logic.delay.DelayManager;
-import org.terasology.logic.delay.PeriodicActionTriggeredEvent;
+import org.terasology.logic.delay.DelayedActionTriggeredEvent;
+import org.terasology.logic.players.LocalPlayer;
 import org.terasology.potions.component.PotionEffect;
 import org.terasology.potions.events.DrinkPotionEvent;
 import org.terasology.registry.In;
 import org.terasology.rendering.nui.NUIManager;
-
-import java.util.ArrayList;
+import org.terasology.rendering.nui.databinding.Binding;
+import org.terasology.rendering.nui.databinding.ReadOnlyBinding;
 
 
 @RegisterSystem(RegisterMode.CLIENT)
 public class PotionStatusUISystem extends BaseComponentSystem {
 
+    private static final String PERIODIC_ACTION_ID = "PotionStatusScreenUI";
+    private static final String TARGET_SCREEN_NAME = "Core:InventoryScreen";
```

Comment:
```suggestion
    private static final String TARGET_SCREEN_NAME = "Inventory:InventoryScreen";
```

Reply:
weirdly, it doesn't work for me locally if I use `Inventory:InventoryScreen`  ... what is this referring to? 🤔